applications/users/views.py

```diff
@@ -108,6 +124,27 @@ def join_group_page(request, group_id):
 Users APIs
 '''
 
+def _get_user(request):
```

Comment:
We dont need a private function for this functionality. You can directly call the controller function from `user_page` method. 

The reason we had private methods for other operations is that the same code was being repeated in multiple place. There is no need for `_get_user `.